src/auto/about_dialog.rs

```diff
@@ -0,0 +1,219 @@
+        }
+    }
+
+    pub fn set_artists(&self, artists: &[&str]) {
+        unsafe {
+            ffi::gtk_about_dialog_set_artists(self.to_glib_none().0, artists.to_glib_none().0);
+        }
+    }
+
+    pub fn set_authors(&self, authors: &[&str]) {
+        unsafe {
+            ffi::gtk_about_dialog_set_authors(self.to_glib_none().0, authors.to_glib_none().0);
+        }
+    }
+
+    pub fn set_comments(&self, comments: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_comments(self.to_glib_none().0, comments.to_glib_none().0);
+        }
+    }
+
+    pub fn set_copyright(&self, copyright: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_copyright(self.to_glib_none().0, copyright.to_glib_none().0);
+        }
+    }
+
+    pub fn set_documenters(&self, documenters: &[&str]) {
+        unsafe {
+            ffi::gtk_about_dialog_set_documenters(self.to_glib_none().0, documenters.to_glib_none().0);
+        }
+    }
+
+    pub fn set_license(&self, license: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_license(self.to_glib_none().0, license.to_glib_none().0);
+        }
+    }
+
+    pub fn set_license_type(&self, license_type: License) {
+        unsafe {
+            ffi::gtk_about_dialog_set_license_type(self.to_glib_none().0, license_type);
+        }
+    }
+
+    pub fn set_logo(&self, logo: Option<&gdk_pixbuf::Pixbuf>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_logo(self.to_glib_none().0, logo.to_glib_none().0);
+        }
+    }
+
+    pub fn set_logo_icon_name(&self, icon_name: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_logo_icon_name(self.to_glib_none().0, icon_name.to_glib_none().0);
+        }
+    }
+
+    pub fn set_program_name(&self, name: &str) {
+        unsafe {
+            ffi::gtk_about_dialog_set_program_name(self.to_glib_none().0, name.to_glib_none().0);
+        }
+    }
+
+    pub fn set_translator_credits(&self, translator_credits: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_translator_credits(self.to_glib_none().0, translator_credits.to_glib_none().0);
+        }
+    }
+
+    pub fn set_version(&self, version: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_version(self.to_glib_none().0, version.to_glib_none().0);
+        }
+    }
+
+    pub fn set_website(&self, website: Option<&str>) {
+        unsafe {
+            ffi::gtk_about_dialog_set_website(self.to_glib_none().0, website.to_glib_none().0);
+        }
+    }
+
+    pub fn set_website_label(&self, website_label: &str) {
+        unsafe {
+            ffi::gtk_about_dialog_set_website_label(self.to_glib_none().0, website_label.to_glib_none().0);
+        }
+    }
+
+    pub fn set_wrap_license(&self, wrap_license: bool) {
+        unsafe {
+            ffi::gtk_about_dialog_set_wrap_license(self.to_glib_none().0, wrap_license.to_glib());
+        }
+    }
+
```

Comment:
There is an extra empty line in every struct by the way (keep me this error, I'll fix it on the generator directly! :D ).